src/sass/variables.scss

```diff
@@ -0,0 +1,12 @@
+@charset "UTF-8";
```

Comment:
How do y'all feel about `globals`? It's not a great name either, but at least there's not a name confusion... :man_shrugging: